pkg/kubernetes/kubernetes.go

```diff
@@ -68,6 +71,9 @@ func (k *Kubernetes) Reconcile(raw map[string]interface{}, objectspecs ...*regex
 		if k != nil && !m.Has("metadata.namespace") {
 			m.Set("metadata.namespace", k.Spec.Namespace)
 		}
+		if k != nil && !m.Has("metadata.labels.tanka/origin") {
```

Comment:
the label name should be defined as a constant.  Also, why are we using "origin", when everywhere else in this code we call it "environment"?